app/controllers/two_factor_authentication/options_controller.rb

```diff
@@ -82,5 +82,12 @@ def build_url(selection, options)
     def two_factor_options_form_params
       params.fetch(:two_factor_options_form, {}).permit(:selection)
     end
+
+    def analytics_properties
```

Comment:
Could these live in a `TwoFactorLoginOptionsForm#extra_analytics_events` for consistency with forms classes being responsible for compiling analytics attributes and keeping the controller lean?

Reply:
Definitely. I initially had it in form, but I was getting a `FakeAnalytics::PiiDetected` error, so I tried it here.